### ISSUE TITLE
Facilitate adding the annotation on federated object to specify users intent of orphaning/not orphaning target/managed resources.

### ISSUE THREAD

**Issue by irfanurrehman** [kubernetes-sigs/kubefed#690]
**What would you like to be added**:
https://github.com/kubernetes-sigs/federation-v2/pull/669 moves away from the usage of `orphanDependents` and uses intent via an annotation on the object to signify whether the dependent resources should be orphaned or not.
This seems to be an additional step, where user will need to create the annotation on the federated object if a non default behaviour is needed.

Suggestion is to facilitate adding the annotation using `kubefed2` before deletion using `kubectl`.
Implement `kubefed2 enable-orphaning-deletion <federated-resource-name>`

**Why is this needed**:

Ease of usage to the user.

/kind feature

**Comment by irfanurrehman** [comment]
A more suitable name then `enable-orphaning-deletion` for the command is welcome.

**Comment by roytman** [comment]
If nobody is working on it, I'll do it

**Comment by marun** [comment]
@roytman Hi!  You're welcome to work on this.  The suggested form would be more like:

```
kubefedctl orphaning-deletion <enable|disable> <resource type> <resource name> [-n namespace]
```

Alternately, you could look at one of the higher-priority items in the [v0.2.0](https://github.com/kubernetes-sigs/kubefed/milestone/5) (beta2) milestone.

**Comment by marun** [comment]
Fixed by #1008
